### PR TITLE
b_reed/APPEALS-15886-v2 

### PR DESCRIPTION
**Pull request by breedbah**
-Updated list with new document types
-Created unit tests in document_types_spec.rb

### CODE REVIEW

File: spec/document_types_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require "spec_helper"
+require "pry"
+describe Caseflow::DocumentTypes do
```
Comment (ThorntonMatthew) [inline spec/document_types_spec.rb:3]: Screenshot from 2023-04-12 11-00-54

I think `require "document_types"` will need to be added to spec_helper.rb in order for the module to be imported to this spec file.

Comment (breedbah) [reply spec/document_types_spec.rb:3]: You are right, I forgot this when I copy pasted code over. Fixing this now

Comment (ThorntonMatthew) [reply spec/document_types_spec.rb:3]: Thank you so much!

File: app/models/caseflow/document_types.rb [hunk 2]
```diff
@@ -282,7 +281,7 @@ module DocumentTypes
       481 => "FOIA/Privacy Act Request",
       482 => "Status Letter",
       483 => "VCAA CheckList for Development",
-      484 => "CRCP Excel Calculation",
+      484 => "CRDP Excel Calculation",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:284]: Nice catch!!!

File: app/models/caseflow/document_types.rb [hunk 3]
```diff
@@ -322,7 +321,7 @@ module DocumentTypes
       521 => "Improved Pension Election Letter",
       522 => "Due Process Waiver",
       523 => "Legal Will",
-      525 => "VA 21-0538 Mandatory Status of Dependents",
+      525 => "VA 21-0538 Mandatory Status of Dependents ",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:324]: I think even though the VBMS doc types have these extra spaces, I think we can "chomp" them off. Curious to what your thoughts are.

Comment (breedbah) [reply app/models/caseflow/document_types.rb:324]: Yeah, I camn take care of that, wasnt sure if there was a reason for it or not

File: app/models/caseflow/document_types.rb [hunk 4]
```diff
@@ -507,7 +506,7 @@ module DocumentTypes
       740 => "VA Form 28-0588, Vocational Rehabilitation and Employment-Getting Ahead After You Get Out",
       741 => "VA Form 28-0791, Preliminary Independent Living Assessment",
       742 => "VA Form 28-1902k, Counseling Record-Education and Vocational Counseling Results",
-      743 => "VA Form 28-1905m, Request for and Receipt of Supplies",
+      743 => "VAF 28-1905m - Request and Authorization for Supplies (Chapter 31 -Veteran Readiness and Employment)",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:509]: 🚀

File: app/models/caseflow/document_types.rb [hunk 5]
```diff
@@ -764,7 +762,7 @@ module DocumentTypes
       1008 => "VAF 29-541 Certificate Showing Residence And Heir",
       1010 => "Award Maintenance",
       1011 => "Death Claim Correspondence",
-      1012 => "Returned mail",
+      1012 => "Award Returned mail",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:765]: I noticed that Award had a strikethrough in the document type doc. I'm thinking we may want to leave it out for now.
1012-award

File: app/models/caseflow/document_types.rb [hunk 6]
```diff
@@ -973,7 +971,7 @@ module DocumentTypes
       1219 => "B&O",
       1220 => "B&O Development",
       1221 => "B&O Folder",
-      1222 => "Death Claim",
+      1222 => "Informal Death Claim",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:974]: 1222-informal

Strikethrough is here as well.

File: app/models/caseflow/document_types.rb (hunk 5 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:765]: Code adjusted

File: app/models/caseflow/document_types.rb (hunk 6 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:974]: Code adjusted

File: app/models/caseflow/document_types.rb [hunk 7]
```diff
@@ -56,7 +56,7 @@ module DocumentTypes
       73 => "Notice of Disagreement",
       79 => "Personnel Record",
       80 => "Photographs",
-      82 => "PHS 2709 Report of Death of Commissioned Officer",
+      82 => "PHS 2709  Report of Death of Commissioned Officer",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:59]: 👾

File: app/models/caseflow/document_types.rb [hunk 8]
```diff
@@ -399,7 +398,7 @@ module DocumentTypes
       601 => "Accounting Approval Letter",
       602 => "Accounting Audit Worksheet",
       603 => "Accounting Estate Breakdown",
-      604 => "Accounting Rejection Letter",
+      604 => "Accounting Rejection Letter ",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:401]: 👾

File: app/models/caseflow/document_types.rb [hunk 9]
```diff
@@ -412,7 +411,7 @@ module DocumentTypes
       615 => "Direct Deposit Enrollment Request Fiduciary - Voided Check",
       616 => "Expense Request - Denial",
       617 => "Fiduciary Appeal - Additional Information Required Letter",
-      618 => "Fiduciary Appeal - Case forwarded to Board of Veterans Appeals",
+      618 => "Fiduciary Appeal - Case  forwarded to Board of Veterans Appeals",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:414]: 👾

File: app/models/caseflow/document_types.rb [hunk 10]
```diff
@@ -458,7 +457,7 @@ module DocumentTypes
       663 => "Medical Opinion",
       664 => "CUE Related",
       665 => "Motion to Advance on Docket",
-      666 => "Motion for Reconsideration",
+      666 => "Motion for Reconsideration ",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:460]: 👾

File: app/models/caseflow/document_types.rb [hunk 11]
```diff
@@ -663,7 +662,7 @@ module DocumentTypes
       907 => "VR-33 Referral - Employment Services",
       908 => "VR-35 Discontinuance Letter - Plan Developed",
       909 => "VR-36 Discontinuance Letter - Failure to Cooperate-Unsatisfactory Conduct",
-      910 => "VR-39 Rehabilitation Letter - Employment",
+      910 => "VR-39 Rehabilitation  Letter - Employment",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:665]: 👾

File: app/models/caseflow/document_types.rb [hunk 12]
```diff
@@ -672,7 +671,7 @@ module DocumentTypes
       916 => "VR-47 Proposed Closure for Non-Pursuit of Claim",
       917 => "VR-48 Proposed Discontinuance - Maximum Rehabilitation Gain",
       918 => "VR-49 Proposed Rehabilitation Letter - Further Education",
-      919 => "VR-50 Discontinuance Letter - Maximum Rehab Gain",
+      919 => "VR-50 Discontinuance Letter - Maximum Rehab Gain ",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:674]: 👾

File: app/models/caseflow/document_types.rb [hunk 13]
```diff
@@ -729,7 +728,7 @@ module DocumentTypes
       973 => "VA Form 21-0789 Your Rights to Representation and a Hearing",
       974 => "VA Form 21-0790 Your Rights to Representation and a Hearing (Possible Overpayment)",
       975 => "VA Form 21-8760 Additional Overpayment for Veterans with Service-Connected Permanent and Total Disability",
-      976 => "VA Form 21-8760a Additional Information for Veterans with Service Connection Under 38 U.S.C 1151 for Permanent and Total Disability",
+      976 => "VA Form 21-8760a  Additional Information for Veterans with Service Connection Under 38 U.S.C 1151 for Permanent and Total Disability",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:731]: 👾

File: app/models/caseflow/document_types.rb [hunk 14]
```diff
@@ -923,10 +921,10 @@ module DocumentTypes
       1169 => "VA Form 28-1904 Agreement to Train OJT Disabled Veterans",
       1170 => "VA Form 28-1917 Monthly Statement of Wages paid to Trainee",
       1171 => "VA Form 28-8739a Protection of Privacy",
-      1172 => "VA Form 28-8832 Application for Counseling",
+      1172 => "VAF 28-8832 - Personalized Career Planning and Guidance Application",
       1173 => "VA Form 2237 Request, Turn in and Receipt for Property of Services-Purchase Request",
       1174 => "VA Form 5655 Financial Status Report",
-      1175 => "VA Form 21-0304 Application for Benefits for Certain Children with Disabilities Born of Vietnam and Certain Korea Service Veterans",
+      1175 => "VA Form 21-0304  Application for Benefits for Certain Children with Disabilities Born of Vietnam and Certain Korea Service Veterans",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:927]: 👾

File: app/models/caseflow/document_types.rb (hunk 7 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:59]: done

File: app/models/caseflow/document_types.rb [hunk 15]
```diff
@@ -1060,7 +1058,7 @@ module DocumentTypes
       1308 => "Camp Lejeune - VHA",
       1309 => "CHAMPVA - VHA",
       1310 => "Clothing Allowance - VHA",
-      1311 => "Eligibility & Enrollment - VHA",
+      1311 => "Eligibility & Enrollment - VHA ",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:1061]: 👾

File: app/models/caseflow/document_types.rb (hunk 8 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:401]: done

File: app/models/caseflow/document_types.rb (hunk 11 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:665]: done

File: app/models/caseflow/document_types.rb (hunk 12 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:674]: done

File: app/models/caseflow/document_types.rb [hunk 16]
```diff
@@ -1082,7 +1080,475 @@ module DocumentTypes
...
+      1619 => "OIF/OEF VONAPP CH35 ORIGINAL CLAIMS",
+      1620 => "CHAPTER 34 CONVERSION-C",
+      1621 => "CHAPTER 34 CONVERSION-A",
+      1622 => "CHAPTER 34 CONVERSION-D",
+      1623 => "EDUCATION GENERAL CERTIFICATION",
+      1624 => "BENEFIT PAYMENT INQUIRY",
+      1625 => "PENDING ISSUE",
+      1626 => "BACK END TOOL - CH33",
+      1627 => "FOLDER FLASH MESSAGE",
+      1628 => "NOTES - OLD FILENET FORMAT ",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:1367]: 👾

File: app/models/caseflow/document_types.rb [hunk 17]
```diff
@@ -1082,7 +1080,475 @@ module DocumentTypes
...
+      1723 => "EVA Form 01 Veteran Financial Documentation",
+      1724 => "EVA Form 02 Veteran Employment Documentation",
+      1725 => "EVA Form 04 VRE Concurrence Documents",
+      1726 => "EVA Form 05 VRE Subsistence Allowance",
+      1727 => "EVA Form 06 VRE Case Management",
+      1728 => "EVA Form 07 VA ONCE Certification",
+      1729 => "EVA Form 08 LTS",
+      1730 => "Appendix Y - Existing Business and Self-Employment Overview",
+      1731 => "Appendix AX - Chapter 31 and Chapter 33 Benefits Comparison",
+      1732 => "Appendix CX - VR&E Officer Concurrence – Ch31 Retroactive   Induction",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:1471]: 👾

File: app/models/caseflow/document_types.rb [hunk 18]
```diff
@@ -1082,7 +1080,475 @@ module DocumentTypes
...
+      1496 => "Request for Application AMA Review Letter",
+      1497 => "Fugitive Felon Due Process Letter",
+      1498 => "Request for Application Compensation Pension or DIC Letter",
+      1499 => "Fugitive Felon Final Notification Letter",
+      1500 => "Non-BDD Claim- Additional Contentions Submitted Letter",
+      1501 => "Non-BDD Claim- Less than 90 Day Letter",
+      1502 => "Non-BDD Claim- Request Resubmission Letter",
+      1503 => "POA Not of Record Letter",
+      1504 => "Third Party Authorization Incorrect Letter",
+      1505 => "HLR Not Timely  Letter",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:1255]: 👾

File: app/models/caseflow/document_types.rb [hunk 19]
```diff
@@ -1082,7 +1080,475 @@ module DocumentTypes
...
-      1757 => "BVA Case Notifications"
+      1333 => "Records Research Center Response",
+      1334 => "Debt Disagreement/Dispute",
+      1335 => "VAF 28-1902f Feasibility Determination Narrative Report",
+      1336 => "VAF 28-1905d Special Report of Training",
+      1337 => "VA Form 26-8497 - Request for Verification of Employment",
+      1338 => "VA Form 26-8497a - Request for Verification of Deposit",
+      1339 => "VA Form 26-8736 - Application For Authority To Close Loans On An Automatic Basis - Non Supervised Lenders",
+      1340 => "VA Form 26-8736A - Non Supervised Lender's Nomination and Recommendation of Credit Underwriter",
+      1341 => "VA Form 26-6381 - Application For Assumption Approval And/Or Release From Personal Liability  To The Government On A Home Loan",
```
[the remark this answers is not in the document]
Comment (breedbah) [reply app/models/caseflow/document_types.rb:1091]: done

File: app/models/caseflow/document_types.rb (hunk 18 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:1255]: done

File: app/models/caseflow/document_types.rb (hunk 16 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:1367]: done

File: app/models/caseflow/document_types.rb [hunk 20]
```diff
@@ -1082,7 +1080,475 @@ module DocumentTypes
...
+      1786 => "VA Form 26-6379 - Progress Sheet on App. for Release From Liability or Sub. of Entitlement (F)",
+      1787 => "VA Form 26-6393 - Loan Analysis",
+      1788 => "VA Form 26-6681 - Application for Fee or Roster Personnel Designation",
+      1789 => "VA Form 26-6704 - Sales Listing-Loan Guaranty Division",
+      1790 => "VA Form 26-6705 - Offer to Purchase and Contract of Sale",
+      1791 => "VA Form 26-6705b - Credit Statement of Prospective Purchaser",
+      1792 => "VA Form 26-6705d - Addendum to Offer to Purchase and Contract of Sale",
+      1793 => "VA Form 26-6705e - Notice of Possible Lead-Based Paint and/or Lead-Based Paint Hazards",
+      1794 => "VA Form 26-6713 - Summary of Basis for Liquidation Procedure",
+      1795 => "VA Form 26-6807 - Financial Statement ",
```
[the remark this answers is not in the document]
Comment (breedbah) [reply app/models/caseflow/document_types.rb:1522]: done

File: app/models/caseflow/document_types.rb [hunk 21]
```diff
@@ -840,7 +838,7 @@ module DocumentTypes
       1086 => "VAF 29-320 Request For Calculation",
       1087 => "VAF 29-352 Application For Rein/Lapsed 6+ Mos",
       1088 => "VAF 29-353 APPLICATION FOR REINSTATEMENT",
-      1089 => "VAF 29-369 Notice of Payment Due - Manual",
+      1089 => "VAF 29-369 Notice of Payment Due - Manual ",
```
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:841]: 👾

File: app/models/caseflow/document_types.rb (hunk 3 above)
Comment (ThorntonMatthew) [inline app/models/caseflow/document_types.rb:324]: 👾

File: app/models/caseflow/document_types.rb (hunk 21 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:841]: done

File: app/models/caseflow/document_types.rb (hunk 3 above)
Comment (breedbah) [reply app/models/caseflow/document_types.rb:324]: done

**Review by ThorntonMatthew — APPROVED** [review]
LGTM!!! 👾 🚀

**Pull request merged**